src/components/Button/README.md

```diff
@@ -321,15 +321,17 @@ function DisclosureButtion() {
 Use when there is only one primary action but other related actions can be taken.
 
 ```jsx
-<Button
-  primary
-  connectedDisclosure={{
-    accessibilityLabel: 'Other save actions',
-    actions: [{content: 'Save as draft'}],
-  }}
->
-  Save
-</Button>
+<div style={{height: '250px'}}>
```

Comment:
Adding the wrapper div to fix the playground not being large enough to accommodate the popover.